health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/BoundaryRelationCreate.js

```
@@ -0,0 +1,682 @@
+
+const BoundaryRelationCreate = () => {
+    const { t } = useTranslation();
+    const location = useLocation();
+    const history = useHistory();
+    const tenantId = Digit.ULBService.getCurrentTenantId();
+    const searchParams = new URLSearchParams(location.search);
+    const hierarchyType = searchParams.get("hierarchyType");
+    const defaultHierarchyType = searchParams.get("defaultHierarchyType")
+    const newHierarchy = JSON.parse(searchParams.get("newHierarchy"));
+    const [showFinalPopup, setShowFinalPopup] = useState(false);
+    let locale = Digit?.SessionStorage.get("initData")?.selectedLanguage || "en_IN";
```

Biome

[error] This let declares a variable that is only assigned once.

'locale' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)